gradle.properties

```diff
@@ -18,3 +18,5 @@ org.gradle.jvmargs=-Xmx2048m -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemor
 # org.gradle.parallel=true
 
 android.useAndroidX=true
+
+android.nonFinalResIds=false
```

Comment:
I used it to make as few code changes as possible, though maybe we should handle this properly. Thoughts?